### ISSUE TITLE
Questions about the conclusion  in the training process and evaluation process #2

### ISSUE THREAD

**Issue by oiskyu** [cs-mshah/CDUL#3]
I checked your project on https://wandb.ai/manan-shah/CDUL , but I did not find the final result of 88.6 mAP that was reported in the paper. The results you uploaded to W&B are similar to what I obtained locally, and neither of them reach 88.6. Could you provide more details about how the 88.6 mAP on the VOC2012 test set was achieved? Specifically, information about the exact hyperparameters, pseudo-label generation settings, training schedule, snippet configuration, and the evaluation/submission process would be extremely helpful for reproducing the reported performance

**Comment by cs-mshah** [comment]
I believe that was the whole point of trying to reproduce. Even we were not able to reproduce the results of the original paper. We however did make some initial mistakes in interpreting the meaning of what is a "snippet size". That was later added to the repo under the `Note` section. Even though the snippet size meaning was understood, we weren't able to reproduce the results from the second stage.  

There is one more mistake in this repository which is the loss function that we used. We should actually be using BCE loss with sigmoid activation instead of softmax activation. More details are in this issue: https://github.com/cs-mshah/CDUL/issues/2.

However, even after these corrections they seem to not have reproduced the exact results. You could try different hparams after the loss corrections and let me know if you are able to reproduce. This is quite concerning since the original authors never released the code, nor was anyone able to reproduce the exact results.
